src/pages/developers/tbtc-security-model.md

```diff
@@ -0,0 +1,126 @@
+2. The Bitcoin price feed, which is used to determine the collateralisation of signers on the Ethereum host chain.
+3. The Bitcoin relay, which verifies BTC transactions to mint and redeem TBTC.
+
+## A decentralized network of signers
+
+tBTC uses the [Keep Network](https://keep.network/) to provide an always-online, decentralized network of signers. Each signer group that custodies a bitcoin deposit is made up of 3 nodes. These nodes are randomly selected from a large pool, using the [Keep random beacon](https://docs.keep.network/random-beacon/). Any opportunity to manipulate the signer set is mitigated by the KEEP token’s staking model, which provides Sybil resistance.
+
+To ensure users are compensated in the case of failure or fraud, signers are overcollateralized in ETH to a ratio of 150% the value of the bitcoin deposit. tBTC handles signer failure and signer fraud, punishing signers and fully reimbursing the user in TBTC, as well as making the signer ineligible for future work.
+
+### Signer failure
+
+Signers not responding to a redemption request in a timely manner constitutes a signer failure. Signer failures can include a failure to produce a signature for redemption, as well as failure to produce an SPV proof of inclusion in a Bitcoin block for a redemption.
+
+All failures are treated as protocol aborts, and initiate the [abort liquidation flow](https://docs.keep.network/tbtc/#redemption). Signer bonds are seized and auctioned off for TBTC so that the user can be reimbursed for their inaccessible funds, and half of the remainder of the signer bonds (a maximum of 1/6th of the original bond) are returned to the signers.
+
+In order to recoup their losses from liquidation, signing groups may choose to recover BTC after an abort has commenced. The reference client leaves offline coordination to each signer.
+
+### Signer fraud
+
+Signer fraud is the only action in the tBTC system that leads to full, punitive slashing. Any signer that moves bitcoin without authorization by the tBTC protocol is slashed, burning their work token and seizing their entire bond for user recourse.
+
+Fraud is proven by presenting the unauthorized signature on-chain as a fraud proof. Anyone can present a fraud proof and receive the ETH left over after auctioning of signer bonds.
+
+Signer fraud is punished in both tBTC and Keep ECDSA protocols.
+
+1. At the tBTC layer, the fraud liquidation flow is enacted and bonds are used to purchase TBTC. The holder of the [tBTC Deposit Token](https://tbtc.network/developers/tbtc-technical-system-overview) is compensated in TBTC, unless the deposit is backing currently-circulating TBTC. In this case, the TBTC is burnt to maintain the 1:1 supply peg.
+2. At the Keep layer, the signer is slashed and their KEEP tokens are burned, potentially removing them from the candidate pool. Slashed signers are not only losing their capital, but opportunity for future fees.
```

Comment:
why are the "KEEP tokens are burned" instead of auctioned?

Reply:
You actually want widespread attacks to result in a loss of liquidity. If the tokens are put back on the market liquidity increases.

The base Keep protocol was also designed two full years before tBTC. If KEEP were only serving tBTC forever, some sort of auction for TBTC might make sense. But KEEP stakers will be able to participate in non-tBTC work down the road, so I still think this is the right call.